packages/react-core/src/demos/DataListDemo.md

```diff
@@ -418,3 +426,353 @@ class ExpandableDataList extends React.Component {
   }
 }
 ```
+
+### Data List view
+
+```js isFullscreen
+import React from 'react';
+import {
+  Badge,
+  Button,
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  Dropdown,
+  DropdownItem,
+  Flex,
+  FlexItem,
+  MenuToggle,
+  MenuToggleCheckbox,
+  OverflowMenu,
+  OverflowMenuControl,
+  OverflowMenuItem,
+  PageSection,
+  PageSectionVariants,
+  Pagination,
+  Text,
+  TextContent,
+  TextVariants,
+  Toolbar,
+  ToolbarItem,
+  ToolbarFilter,
+  ToolbarContent,
+  Select
+} from '@patternfly/react-core';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
+
+import CodeBranchIcon from '@patternfly/react-icons/dist/esm/icons/code-branch-icon';
+import CodeIcon from '@patternfly/react-icons/dist/esm/icons/code-icon';
+import CubeIcon from '@patternfly/react-icons/dist/esm/icons/cube-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-triangle-icon';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+import TimesCircleIcon from '@patternfly/react-icons/dist/esm/icons/times-circle-icon';
+
+class DataListViewBasic extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      page: 1,
+      perPage: 5,
+      totalItemCount: 5
+    };
+  }
+
+  renderPagination() {
+    const { page, perPage, totalItemCount } = this.state;
```

Comment:
`perPage` is not used by this function anymore. I wonder if it's still needed in the state at all?